### DISCUSSION TITLE
使用telegram推送时，出现400错误

### DISCUSSION THREAD

**Discussion by jiewenhuang** [walinejs/waline#1698] [Q&A] [answered]
具体：
Telegram Notification Failed:{"ok":false,"error_code":400,"description":"Bad Request: can't parse entities: Character '#' is reserved and must be escaped with the preceding '\\'"}
使用的默认模板，这个应该怎么解决呢？

**Comment by lizheming** [comment, marked as answer]
消息内容里包含一些字符串 telegram 无法解析。建议修改模板将其替换成普通文案，例如「你有一条新评论」之类的固定内容
